soen341-UB3-QA/resources/views/ask.blade.php

```diff
@@ -36,21 +36,26 @@
             <!-- Comments Form -->
             <div class="card my-4">
                 <div class="card-body">
-                    <form>
+                    {{--<form>--}}{!! Form::open(['url' => 'ask']) !!}
                         <div class="form-group">
-                            <h4 class="mt-4">Title:</h4>  
-                            <textarea class="form-control" rows="1"></textarea>
-                        </div> 
-                    </form>
+                            {!! Form::label('title','Title:') !!}
+                            {!! Form::text('title', null, ['class' => 'form-control', 'placeholder' => 'Titles...']) !!}
```

Comment:
"Titles" is a bad placeholder value. Try something like "Enter your question title here"